src/test/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepositoryTest.java

```diff
@@ -39,7 +39,7 @@
 
     @BeforeEach
     void setUp() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.initMocks(this); // replace initMocks with OpenMocks
```

Comment:
Either action or remove these comments if not required for Sonar?

Reply:
Updated

Reply:
Updated , removed commented statements